code/modules/awaymissions/zlevel.dm

```diff
@@ -6,7 +6,13 @@
 		air_master.setup_allturfs(turfs)
 	for(var/turf/T in turfs)
 		if(T.dynamic_lighting)
-			T.lighting_build_overlays()
+			T.lighting_build_overlay()
+		T.lighting_corners_initialised = TRUE
```

Comment:
I'm still serious when I say that this shouldn't be handled in away mission code.

Any map loading will break extremely badly and you're just band aiding it in this specific instance.

Changeturf exists for a reason. This band aiding makes the code incredibly unmaintainable AND it's not even doing `recalc_atom_opacity()`.

If lag is actually an issue due to Linda, then 2 things:
* \>LINDA is slower than ZAS at changeTurf, kek.
* Add an argument to changeTurf to only do lighting stuff.